FlightReservation/App.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ * @flow strict-local
+ */
+
+import React, { useState } from 'react';
+import {
+  SafeAreaView,
+  StatusBar,
+} from 'react-native';
+
+import MainScreen from './components/mainScreen';
```

Comment:
it would be better to have a folder with the `screens` and a folder with the `components`.

Reply:
thank you for your suggestions, we already applied them, check it on the next pull request.